website/docs/r/autoscaling_policy.html.markdown

```diff
@@ -49,6 +49,10 @@ The following arguments are supported:
 * `policy_type` - (Optional) The policy type, either "SimpleScaling", "StepScaling" or "TargetTrackingScaling". If this value isn't provided, AWS will default to "SimpleScaling."
 * `estimated_instance_warmup` - (Optional) The estimated time, in seconds, until a newly launched instance will contribute CloudWatch metrics. Without a value, AWS will default to the group's specified cooldown period.
 
+The following argument is only available to "SimpleScaling" and "StepScaling" type policies:
+
+* `min_adjustment_magnitude` - (Optional) Minimum number of instances to scale when `adjustment_type` is set to "PercentChangeInCapacity".
```

Comment:
Just a minor suggestion to closer match the [API Reference](https://docs.aws.amazon.com/autoscaling/ec2/APIReference/API_PutScalingPolicy.html) and use backtick formatting with the specific value:

```suggestion
* `min_adjustment_magnitude` - (Optional) Minimum value to scale by when `adjustment_type` is set to `PercentChangeInCapacity`.
```